File: app/src/main/java/app/labs14/roamly/localStorage/trip/TripDbContract.java
package app.labs14.roamly.localStorage.trip;

import android.provider.BaseColumns;

public class TripDbContract {
    public static class TripEntry implements BaseColumns {
        public static final String TABLE_NAME                  = "trips";
        public static final String COLUMN_NAME_NAME  = "name";


        public static final String SQL_CREATE_TABLE = "CREATE TABLE " + TABLE_NAME
                + " ( " +
                _ID + " INTEGER PRIMARY KEY," +
                COLUMN_NAME_NAME + " TEXT" +
                "  );";

        public static final String SQL_DELETE_TABLE = "DROP TABLE IF EXISTS " + TABLE_NAME;
    }
}

File: app/src/main/java/app/labs14/roamly/localStorage/TripDBContract.kt
package app.labs14.roamly.localStorage

import android.provider.BaseColumns

class TripDBContract {
    class TripEntry : BaseColumns {
        companion object {
            const val TABLE_NAME = "trips"
            const val COLUMN_NAME_IS_READ = "is_read"
            const val _ID  = ""

            const val SQL_CREATE_TABLE = ("CREATE TABLE " + TABLE_NAME
                    + " ( " +
                    BaseColumns._ID + " INTEGER PRIMARY KEY," +
                    COLUMN_NAME_IS_READ + " INTEGER" +
                    "  );")

            const val SQL_DELETE_TABLE = "DROP TABLE IF EXISTS $TABLE_NAME"
        }
    }
}
